SteakBot.Core/Objects/Enums/MemeResultType.cs

```diff
@@ -1,7 +1,8 @@
 ﻿namespace SteakBot.Core.Objects.Enums
 {
-    internal enum MemeResultType
+    public enum MemeResultType
```

Comment:
Enums are bad design when used as a branching mechanism, refactor!

Reply:
This is not part of this PR, so feel free to refactor in a follow-up.